package.json

```diff
@@ -73,6 +74,7 @@
     "jest": "^18.1.0",
     "jest-cli": "^17.0.3",
     "jest-react-native": "^17.0.3",
+    "react-native-debugger-open": "^0.3.2",
```

Comment:
Yeah, actually I'll remove it for now as I included a launch command in `package.json` instead.